OptimoveSDK/Sources/Classes/OptimoveConfig.swift

```diff
@@ -161,100 +217,150 @@ open class OptimoveConfigBuilder: NSObject {
         return self
     }
 
-    /**
-     Internal SDK embedding API, do not call or depend on this method in your app
-     */
-    @discardableResult public func setBaseUrlMapping(baseUrlMap: ServiceUrlMap) -> OptimoveConfigBuilder {
-        _baseUrlMap = baseUrlMap
-
-        return self
-    }
-
     @discardableResult public func build() -> OptimoveConfig {
-        var tenantInfo: OptimoveTenantInfo?
-        var optimobileConfig: OptimobileConfig?
-
-        if let _tenantToken = _tenantToken, let _configName = _configName {
-            tenantInfo = OptimoveTenantInfo(tenantToken: _tenantToken, configName: _configName)
-        }
-
-        if let _apiKey = _apiKey,
-           let _secretKey = _secretKey,
-           let _baseUrlMap = _baseUrlMap,
-           let _region = _region
-        {
-            optimobileConfig = OptimobileConfig(
-                apiKey: _apiKey,
-                secretKey: _secretKey,
-                region: _region,
-                sessionIdleTimeout: _sessionIdleTimeout,
-                inAppConsentStrategy: _inAppConsentStrategy,
-                inAppDefaultDisplayMode: _inAppDisplayMode,
-                inAppDeepLinkHandlerBlock: _inAppDeepLinkHandlerBlock,
-                pushOpenedHandlerBlock: _pushOpenedHandlerBlock,
-                _pushReceivedInForegroundHandlerBlock: _pushReceivedInForegroundHandlerBlock,
-                deepLinkCname: _deepLinkCname,
-                deepLinkHandler: _deepLinkHandler,
-                baseUrlMap: _baseUrlMap,
-                runtimeInfo: _runtimeInfo,
-                sdkInfo: _sdkInfo,
-                isRelease: _isRelease
-            )
-        }
+        let tenantInfo: OptimoveTenantInfo? = {
+            if let _tenantToken = _tenantToken,
+               let _configName = _configName
+            {
+                return OptimoveTenantInfo(tenantToken: _tenantToken, configName: _configName)
+            }
+            Logger.info("\(OptimoveTenantInfo.self) building failed.")
```

Comment:
mb it's more like 'skipped' Vs 'failed'. optimove creds can be null . same for below